src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodAboveComments.java

```diff
@@ -0,0 +1,91 @@
+/*
+JavadocMethod
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+public class InputJavadocMethodAboveComments {
+
+    /**
+     * A Javadoc comment.
+     */
+    //@ A JML Annotation
+    public int foo() { // violation, '@return tag should be present and have description.'
```

Comment:
from user perspective it is not clear what is considered as javadoc comment. may be `//@ A JML Annotation` ?

```
    /**
     * A Javadoc comment.
     * @return result
     */
    //@ A JML Annotation
    public int foo() throws Exception { // violation, '@throw tag should be present and have description.'
```

let update to this to give more hints that return is found, so javadoc is second comment from method signature.
same for all others.